src/Websites/Sales/Bit.Websites.Sales/Web/compilerconfig.json.defaults

```diff
@@ -6,7 +6,8 @@
             "style": "expanded",
             "sourceMapUrls": "relative",
             "relativeUrls": true,
-            "sourceMap": false
+            "sourceMap": false,
+            "useNodeSass": false
```

Comment:
What's that `useNodeSass`?

Reply:
it's probably from the webcompiler extension.